### ISSUE TITLE
macvim: crashes after installing from bottle, works after manual compilation

### ISSUE THREAD

**Issue by Kwpolska** [Homebrew/homebrew-core#26699]
- [x] are reporting a bug others will be able to reproduce and not asking a question. If you're not sure or want to ask a question do so on our Discourse: https://discourse.brew.sh
- [x] have a problem with `brew install` (or `upgrade`, `reinstall`) a single, official formula (not cask)? If it's a general `brew` problem please file this issue at Homebrew/brew: https://github.com/Homebrew/brew/issues/new. If it's a `brew cask` problem please file this issue at https://github.com/caskroom/homebrew-cask/issues/new. If it's a tap (e.g. Homebrew/homebrew-php) problem please file this issue at the tap.
- [x] ran `brew update` and can still reproduce the problem?
- [x] ran `brew doctor`, fixed all issues and can still reproduce the problem?  → one issue left over, but it’s in `ghostscript`, which is unrelated
- [x] ran `brew gist-logs <formula>` (where `<formula>` is the name of the formula that failed) and included the output link?
- [x] if `brew gist-logs` didn't work: ran `brew config` and `brew doctor` and included their output with your issue?

`brew gist-logs macvim`: https://gist.github.com/74267b9ab0ecf051b119d5ddf6cc2169
```
$ brew install macvim
$ mvim
Fatal Python error: Py_Initialize: unable to load the file system codec
ModuleNotFoundError: No module named 'encodings'

Current thread 0x00007fff96fd2380 (most recent call first):
Vim: Caught deadly signal ABRT
Vim: Finished.
$ brew reinstall -s macvim
$ mvim
(works properly)
```

**Comment by Kwpolska** [comment]
I confirm that this was fixed by #26722.

**Comment by ilovezfs** [comment]
@Kwpolska thanks for confirming!
